### ISSUE TITLE
Invalid directory error

### ISSUE THREAD

**Issue by irfaan** [holmesal/react-native-ios-notification-actions#1]
Hi! Hitting the following:

```
uncaught error Error: UnableToResolveError: Unable to resolve module react-native-notification-actions from /[path]/index.ios.js: Invalid directory /Users/node_modules/react-native-notification-actions
```

React seems to be looking for the module in `Users/node_modules/react-native-notification-actions`, when it is of course in `[path]/node_modules/react-native-notification-actions`

RN 0.18.0-rc

Any ideas?

Thanks! Looking forward to using (and hopefully contributing) to your module!

**Comment by irfaan** [comment]
Related: https://github.com/facebook/react-native/issues/4968#thread-subscription-status

Unfortunately, the fixes there did not repair the error. Any insight into this @holmesal ? If not, might just close this issue and keep watching for another fix in that thread.

**Comment by irfaan** [comment]
Oops - was referencing the module wrong! Seems to be only related to that issue insofar as the `Users/` directory was the last place up the chain it looked for the module (see https://github.com/facebook/react-native/issues/4968#issuecomment-167662382).

Pushing a slight correction to the docs.

**Comment by holmesal** [comment]
Fixed w/ merge of #2
